src/entity/Human.php

```diff
@@ -211,6 +211,14 @@ public function getXpManager() : ExperienceManager{
 		return $this->xpManager;
 	}
 
+	public function getXpSeed() : int{
+		return $this->xpSeed;
+	}
+
+	protected function setRandomXpSeed() : void{
+		$this->xpSeed = random_int(Limits::INT32_MIN, Limits::INT32_MAX);
```

Comment:
Use `mt_rand` instead of random_int

Reply:
Not really sure why you said this, but the original code used `random_int()` also. I don't really think it makes a difference, but do you have a particular reason for this preference?